templates/zerver/help/zulip-on-windows.md

```diff
@@ -0,0 +1,44 @@
+# Using Zulip on Windows
+
+Zulip has a Windows client, compatible with Windows versions 7, 8 and 10 and is identical in
+functionality to the Zulip Web Application. The client
+can be download through https://zulip.org/.
+
+The client is regularly updated and is a wrapper of the Web App, but with enhanced
+notifications and can run in the background. The current version does not contain
+any significant features additional to those in the Web App.
+
+!!! warn ""
+    **Note:** Zulip is no longer running on a browser, many external keyboard shortcuts
+    endemic to a browser will no longer work, for example, ALT-LEFT to go back to the previous page.
+
+## Initial setup
+### Connecting to a server
+
+When you open the Zulip client for the first time, you will come across a screen
+asking for your organizations's server address on Zulip. Enter your organization's
+address and press connect.
```

Comment:
Is "connect" a button, link, etc.? You should rephrase it to "... Press the `**Connect**` button to connect to your organization's server."